src/SUMMARY.md

```diff
@@ -51,7 +51,7 @@
 
 - [Obsługa błędów](ch09-00-error-handling.md)
     - [Nienaprawialne Błędy z `panic!`](ch09-01-unrecoverable-errors-with-panic.md)
-    - [Recoverable Errors with `Result`](ch09-02-recoverable-errors-with-result.md)
+    - [Możliwe do Odzyskania Błędy z `Result`](ch09-02-recoverable-errors-with-result.md)
```

Comment:
Możliwe do Odzyskania Błędy brzmi niezrozumiale.
Może "naprawialne błędy?"

Reply:
choć "naprawialne błędy" wydają mi się najlepszym tłumaczniem, to też wprowadza ono nieco w błąd, tak na prawdę to są błędy na które można zareagować, ale nie zawsze da się naprawić. Nie rozumiem co miał na myśli autor piszący "Recoverable", może coś w rodzaju 3 znaczenia z:
https://www.diki.pl/slownik-angielskiego?q=Recoverable&source=opensearch

Reply:
Dziwnie brzmi "naprawialne błędy" ale skoro w poprzednim rozdziale mamy "naprawialne błędy" to może tak zostać.

Reply:
Czytając teraz kolejny rozdział "ch09-03-to-panic-or-not-to-panic" myślę, że autorowi chodziło o to, że w przypadku paniki nie ma sposobu na odzyskanie danych a w Result można próbować coś odzyskać.

Reply:
https://pl.bab.la/slownik/angielski-polski/recoverable-error
Zdecydowanie naprawialne, kilka słowników tak podaje.

Reply:
jeśli chodzi o odzyskiwanie jakiś danych to przecież nie chodzi o odzyskiwanie samego błędu, pisanie że błąd jest odzyskiwalny jest skrótem myślowym, który wydaje mi się nie zrozumiały

Reply:
tłumacz z Heliona użył słów odwracalny/nieodwracalny i choć to też, jak pozostałe pomysły, jest dalekie od ideału, to myślę że niegłupio byłoby zachować z nim zgodność by nie wprowadzać zbyt wielu różnych polskich tłumaczeń na to samo
https://helion.pl/pobierz-fragment/programowanie-w-jezyku-rust-oficjalny-podrecznik-wydanie-ii-steve-klabnik-carol-nichols,prruo2/pdf

Reply:
Zmieniłem - wybrałem wersję z Heliona.